catalog/spatial/registry/registry-source-configuration-handler/src/main/java/org/codice/ddf/registry/source/configuration/SourceConfigurationHandler.java

```diff
@@ -315,14 +336,22 @@ private void activateDeactivateExistingConfiguration(String fPidToActivate,
         //Order of disable/enable important. Can't have two enabled configurations with the same
         //id so if there is one to disable, disable it before enabling another one.
         if (configToDisable != null) {
-            fpidToConfigurationMap.remove(configToDisable.getFactoryPid());
+            fpidToConfigurationMap
+                    .remove(configToDisable.getFactoryPid() + configToDisable.getProperties()
```

Comment:
I'll leave it with the + concat, if that's cool. It does the automatic type conversion for the getProperties property (which is why I'm not using concat here)  and is more clear than the String.format. String.format doesn't bring any performance boost or anything.